nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+const app = express();
+const bodyParser = require("body-parser");
+app.use(
+    bodyParser.urlencoded({
+      extended: true,
+    })
+  );
```

Comment:
in latest express versions, you don't have to require the bodyParser, you can just do this:
```
app.use(express.urlencoded({ extended: true }));
```